test/ast.spec.js

```diff
@@ -661,6 +661,12 @@ describe('AST', () => {
                 expect(getParsedSql(sql)).to.equal(`SELECT 1 UNION ALL SELECT '1' UNION SELECT \`a\` FROM \`t\` UNION ALL (SELECT TRUE)`);
             });
 
+            it('should combine multiple statements union distinct', () => {
+              sql = `select 1 union distinct select '1' union select a from t union distinct (select true)`;
+              const ast =
```

Comment:
`const ast=` codes  can be removed

Reply:
Removed by [c7ffc88](https://github.com/taozhi8833998/node-sql-parser/pull/1538/commits/c7ffc88040b2755c77b4367f3c497c5d896bc8aa)